app/src/main/java/com/github/orkest/Model/FireStoreDatabaseAPI.kt

```diff
@@ -168,6 +168,22 @@ class FireStoreDatabaseAPI {
         return future
     }
 
+    /**
+     * @param post the post we want to add in the database
+     * @param
+     * @return a completable future that completes to true only if post is correctly added
+     */
+    fun addCommentInDataBase(username: String, post_date: String, comment: Comment): CompletableFuture<Boolean> {
+        val future = CompletableFuture<Boolean>()
+        val postDocument = getPostCollectionRef(username).document("${post_date}/comments/${comment.date}")
```

Comment:
Maybe you could refactor this line into a getCommentCollectionRef function to improve readability and avoid code duplication